svc-bip-api/src/main/java/gov/va/vro/bip/config/RMQConfig.java

```diff
@@ -0,0 +1,63 @@
+package gov.va.vro.bip.config;
+
+import gov.va.vro.bip.model.HasStatusCodeAndMessage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.support.ListenerExecutionFailedException;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.UUID;
+
+@Configuration
+@Slf4j
+public class RMQConfig {
+  @Value("${exchangeName}")
+  String exchangeName;
+
+  @Bean
+  public MessageConverter jackson2MessageConverter() {
+    return new Jackson2JsonMessageConverter();
+  }
+
+  @Bean
+  DirectExchange bipApiExchange() {
+    return new DirectExchange(exchangeName, true, true);
+  }
+
+  public static Object respondToClientDueToUncaughtExcdeption(
+      Message amqpMessage,
+      org.springframework.messaging.Message<?> message,
+      ListenerExecutionFailedException exception,
+      HasStatusCodeAndMessage rVal)
+      throws Exception {
+    try {
```

Comment:
Unnecessary try-catch here. Nothing inside the body of this method will throw an exception. You can also remove the throws Exception from the method signature.

Reply:
What if `message` is `null`?  

 There is however an unnecessary `throws` on line 36.